### DISCUSSION TITLE
"Open in browser" option

### DISCUSSION THREAD

**Discussion by Gladear** [community/community#136967] [Mobile] [answered]
It's a secret to no one some features are missing from the mobile apps (and I don't even think that's a bad thing, the mobile app is great, I don't want it to be cluttered with features that make no sense on mobile).

However, I sometimes want to access some features on mobile that do not exist. The flow is usually: open the app, browse to a repo, need to do something, open the browser, browse to the repo *again*, do my thing.

I'd like to be able to skip steps "open the browser, browser to the repo again", and to be able to open the repo right from the app. I think a "Open in browser" menu option would be nice to have and simplify some operations.

What do you think about it?

**Comment by eliperkins** [comment, marked as answer]
Hey @Gladear, thanks for this feedback.

> It's a secret to no one some features are missing from the mobile apps (and I don't even think that's a bad thing, the mobile app is great, I don't want it to be cluttered with features that make no sense on mobile).

💯  same page as you. Part of our team's philosophy in designing the app has been to add an "escape hatch" to get to the browser, for any needs beyond what the app can provide. In most cases, this manifests as a Share button, sometimes within the ••• overflow menu in navigation bars. Each screen within GitHub Mobile _should_ have a Share option that will let you open in the browser of your choice.

If you're not seeing that present on any specific screen, that's an oversight on our part! Please let us know if that's the case.

Otherwise, let me know what you think about a Share button being the way to escape out of GitHub Mobile and into the browser! I'd love to hear your feedback on this.

**Reply by Gladear** [reply]
Oh ok, I didn't think about the share button to open in browser, that's not how I use it usually 😄
Thanks for the answer, that works fine 👌

**Reply by eliperkins** [reply]
Glad to hear that this approach solves the need! 

> I didn't think about the share button to open in browser, that's not how I use it usually

To help us improve, where might you expect to see this instead? What are some other alternative patterns we could explore to make this easier?

**Reply by Gladear** [reply]
I would probably have expected an "Open in browser" button, probably next to the "Share" one in the three-dots menu, I think that would make it very explicit 😄
